src/main/java/io/nats/client/api/MessageGetRequest.java

```diff
@@ -52,6 +54,8 @@ public static byte[] seqBytes(long sequence) {
 
     /**
      * @deprecated use static method lastForSubject with .serialize instead
+     * @param subject filter subject
+     * @return rendered output
```

Comment:
I personally hate having comments on deprecated api because it might encourage their use. It might be good to have proper links to the replacement though. There are a couple items like this.

Reply:
I know, I just wanted to get rid of the warnings.